### PR TITLE
docs: compress images

### PR DESCRIPTION
**Pull request by XhmikosR**
I guess I should drop the SVG patch since those are being compressed on build time, but we need to sort out #3800 so that we get the best results.

If everyone's happy with the result, I will squash the other 2 PNG patches into 1.

The reason for this patches is that I used the best possible options and tools for PNG images, optipng, zopflipng, pngout and deflopt, with the maximum compression settings.

So, this will get us the best result, and also docs build time should decrease, in theory.

EDIT:

From a quick test I see page size has dropped 30.2 KB with these patches.

### CODE REVIEW

**Comment by XhmikosR** [comment]
From a quick test I see page size has dropped 30.2 KB with these patches.

File: docs/images/mocha-logo.svg [hunk 1]
```diff
@@ -1,125 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (plroebuck) [inline docs/images/mocha-logo.svg]: Don't think you should have deleted the [XML declarations](https://stackoverflow.com/a/38172170).

Comment (XhmikosR) [reply docs/images/mocha-logo.svg]: Of course they can be deleted.

Comment (XhmikosR) [reply docs/images/mocha-logo.svg]: But see my original comment. The production SVGs are compressed with the svgo default atm.

And you will see this stuff is removed from there too.

**Comment by boneskull** [comment]
@Munter 	I'm not sure what you're requesting here... @XhmikosR 	do you understand?

**Comment by XhmikosR** [comment]
Yes, but like I explain, this results in better compression that what the tool you use is doing.

**Comment by XhmikosR** [comment]
I can keep the lossless compression patch, drop SVGH, as long as someone guides me to that tool that's being used image compression option. But even so, we'll never get the same results as this, because I use extensively slow settings for the best savings, that we don't want to have on each build.

**Pull request closed without merging**